nav2_system_tests/src/waypoint_follower/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ ament_add_test(test_waypoint_follower
   GENERATE_RESULT_FOR_RETURN_CODE_ZERO
   COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/test_case_py.launch"
   WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
-  TIMEOUT 100
+  TIMEOUT 180
```

Comment:
When locally I've reproduced a timeout problem for `test_waypoint_follower`, the test is infinitely writing the messages like:
```
[waypoint_follower-11] [INFO] [1590052230.049948970] [waypoint_follower]: Processing waypoint 2...
[waypoint_follower-11] [INFO] [1590052230.099968519] [waypoint_follower]: Processing waypoint 2...
[waypoint_follower-11] [INFO] [1590052230.149970479] [waypoint_follower]: Processing waypoint 2...
[waypoint_follower-11] [INFO] [1590052230.199950262] [waypoint_follower]: Processing waypoint 2...
[waypoint_follower-11] [INFO] [1590052230.249950544] [waypoint_follower]: Processing waypoint 2...
[waypoint_follower-11] [INFO] [1590052230.299922573] [waypoint_follower]: Processing waypoint 2...
[waypoint_follower-11] [INFO] [1590052230.349954831] [waypoint_follower]: Processing waypoint 2...
[waypoint_follower-11] [INFO] [1590052230.399980811] [waypoint_follower]: Processing waypoint 2...
[waypoint_follower-11] [INFO] [1590052230.449920775] [waypoint_follower]: Processing waypoint 2...
...
```
After the following problem appeared in log:
```
[controller_server-7] [ERROR] [1590052228.696053172] [DWBLocalPlanner]: No valid trajectories out of 419!
[controller_server-7] [ERROR] [1590052228.696123844] [DWBLocalPlanner]: 1.00: BaseObstacle/Trajectory Hits Obstacle.
[controller_server-7] [ERROR] [1590052228.696528810] [controller_server]: No valid trajectories out of 419!
```
In other hand, when `test_waypoint_follower` passes, there is no such error messages appeared in log.

I think we need here to make more analysis of root cause of the problem causing above error messages. Maybe raise a new ticket for that, since it seems to be another problem. Increasing of timeout will not fix it, but only increase testing time in CI.